src/API/index.ts

```diff
@@ -1 +1,4 @@
-export const baseUrl= "https://pro-commerce1.herokuapp.com/api/v1"
\ No newline at end of file
+export const baseUrl= "https://pro-commerce1.herokuapp.com/api/v1"
+
+export const token =
+    "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6Im1vc3NhMTIzQGdtYWlsLmNvbSIsIm5hbWUiOiJiaWdzYW0iLCJpYXQiOjE2NjMxNDAxNjl9.WMLMav7L4dlUOjfYfKQx7_JeGcIHP4aRaSX2TlLfCQY";
```

Comment:
Token must not be static, it should be dynamic depending on the current logged in user